website/versioned_docs/version-0.63/cli.md

```diff
@@ -0,0 +1,77 @@
+|-----------------------|------------|--------------------------------------------------|
+| `--release`           | boolean    | Specifies a Release build                        |
+| `--root`              | string     | Override the root directory for the windows build which contains the windows folder. (default: `E:\\test63`) |
+| `--arch`              | string     | The build architecture (`ARM64`, `x86`, `x64`). defaults to system architecture |
+| `--singleproc`        | boolean    | Disables multi-proc build                        |
+| `--emulator`          | boolean    | Deploys the app to an emulator                   |
+| `--device`            | boolean    | Deploys the app to a connected device            |
+| `--target`            | string     | Deploys the app to the specified `GUID` for a device |
+| `--remote-debugging`  | boolean    | Deploys the app in remote debugging mode.        |
+| `--logging`           | boolean    | Enables logging                                  |
+| `--no-packager`       | boolean    | Do not launch packager while building            |
+| `--bundle`            | boolean    | Enable Bundle configuration and it would be `ReleaseBundle`/`DebugBundle` other than `Release`/`Debug` |
+| `--no-launch`         | boolean    | Do not launch the app after deployment           |
+| `--no-autolink`       | boolean    | Do not run autolinking                           |
+| `--no-build`          | boolean    | Do not build the solution                        |
+| `--no-deploy`         | boolean    | Do not deploy the app                            |
+| `--deploy-from-layout`| boolean    | Force deploy from layout, even in release builds |
+| `--sln`               | string     | Override the app solution file determined by 'react-native config', e.g. `windows\myApp.sln` |
+| `--proj`              | string     | Override the app project file determined by 'react-native config', e.g. `windows\myApp\myApp.vcxproj` |
+| `--msbuildprops`      | string     | Comma separated props to pass to MSBuild, e.g. `prop1=value1,prop2=value2` |
+| `--buildLogDirectory` | string     | Optional directory where MSBuild log files should be stored |
+| `--info`              | boolean    | Dump environment information                     |
+| `--direct-debugging`  | number     | Enable direct debugging on specified port        |
+| `--no-telemetry`      | boolean    | Disables sending telemetry that allows analysis of usage and failures of the react-native-windows CLI |
+| `-h`, `--help`        | boolean    | output usage information                         |
+
+This sends telemetry to Microsoft by default. You can prevent the telemetry from being sent by using the `--no-telemetry` command line option. See the `@react-native-windows/cli` [README](https://github.com/microsoft/react-native-windows/edit/main/packages/@react-native-windows/cli/README.md) for more details.
```

Comment:
this link is to the "edit" page on github for the readme.md file, recommend either not linking to it, linking to the raw file, or to the rendered file instead

Reply:
Will do!